.travis.yml

```diff
@@ -65,16 +48,16 @@ jobs:
       go: "1.12.x"
       os: "osx"
     - <<: *test
-      go: "1.x"
+      go: "1.13.x"
       os: "linux"
     - <<: *test
-      go: "1.x"
+      go: "1.13.x"
       os: "osx"
     - <<: *test
-      go: "master"
+      go: "1.x"
       os: "linux"
     - <<: *test
-      go: "master"
+      go: "1.x"
```

Comment:
I think we should keep testing against `master` here too. We are allowing failures against it anyway, so we might as well keep it.

Reply:
Sorry, I now see it's been moved to the top of the list. I think it would be better to keep it at the bottom, both for the logical order, and for people viewing the changeset and thinking that it was removed (like I did!).

Reply:
The reason it was moved is that the first one is tagged for later versions in the config file to reference it. Having the oldest version at the top of the file means touching more lines of code when/if we drop support for a version - however the master config should almost never change, so it makes more sense for that to be tagged as the reference configuration?

Reply:
I think the part where it tags it for later versions is:
```yml
  include:
    - &test
      stage: Test
      go: "master"
      os: "linux"
      before_script:
```
And then this declaration is below that, so we can keep the one above at the top, and move this one back to the bottom, right?

Reply:
Have checked in a commit restoring master to bottom of list to show what I mean.

Reply:
As discussed, we'll leave it like this for now, and will reorder the versions to be most recent at top (master), to oldest at bottom in a separate changeset, so as not to interfere with the changes we've made here.